src/ImageSharp/Formats/Gif/GifDecoderCore.cs

```diff
@@ -185,7 +185,7 @@ public ImageInfo Identify(BufferedReadStream stream, CancellationToken cancellat
     {
         uint frameCount = 0;
         ImageFrameMetadata? previousFrame = null;
-        List<ImageFrameMetadata> framesMetadata = new();
+        List<ImageFrameMetadata> framesMetadata = [];
```

Comment:
This could be enforced by this editorconfig rule https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0028?pivots=dotnet-8-0

Reply:
If you want I can look at enforcing this style in a seperate pr

Reply:
That would be nice thanks. The IDE has been making the suggestion for me, that and range usage also.